src/usr/share/gpgmailer/gpgmailer.py

```diff
@@ -116,6 +120,41 @@ def start_monitoring(self):
                                   str(exception))
                 self.logger.error(traceback.format_exc())
 
+    def _read_and_send_message(self, file_name, loop_start_time):
+        """Attempt to build a message and send it, re-raising the SenderKeyExpired and
+        NoUsableKeysException exceptions.
+        """
+        try:
+            message_dict = self._read_message_file(file_name)
+
+            # Set default subject if the queued message does not have one.
+            if message_dict['subject'] is None:
+                message_dict['subject'] = self.config['default_subject']
+
+            encrypted_message = self._build_encrypted_message(
+                message_dict, loop_start_time)
+
+            self._send_mail(mime_message=encrypted_message,
+                            recipients=self.valid_recipient_emails)
+            self.logger.info('Message %s sent successfully.', file_name)
+
+            os.remove(os.path.join(self.outbox_path, file_name))
+
+        except SendmailError as sendmail_error:
+            self.logger.error('Message %s was not queued: %s',
+                              file_name, str(sendmail_error))
```

Comment:
I don't see any reason to treat a SendmailError any differently in this context.